File: sample/backend/backend_express/jwtUtil.js
const jwt = require('jsonwebtoken');

const generateAccessToken = (employee) => {
    return jwt.sign({
        'username': employee.username,
        'role': employee.role,
        'iat': new Date().getTime(),
        'exp': new Date().getTime() + 30 * 60000},
         process.env.TOKEN_SECRET);
}

module.exports = {
    generateAccessToken
}

File: sample/backend/backend_flask/jwtUtil.py
import jwt
from datetime import datetime, timedelta

def jwtDecode(token, secret, algorithms):
    return jwt.decode(token, secret, algorithms)

def generateToken(employee, secret, algorithm):
    # generates the JWT Token
    return jwt.encode({
        'username': employee.username,
        'role': employee.role,
        'iat' : datetime.utcnow(),
        'exp' : datetime.utcnow() + timedelta(minutes = 30)
    }, secret, algorithm)